### ISSUE TITLE
iFrame consumes 'focusin' event, causing errors with keyboard-outline

### ISSUE THREAD

**Issue by kanterjoe** [campuspress/divi-accessibility#25]
When a keyboard user tabs through the page, `.keyboard-outline` class is applied to the selected item. This class must be removed from the the previously selected item as well. However, when there is an iframe (such as an embedded youtube video) in the tab order, this does not work. This is because the removal relies on the next item triggering a `focusin` event. When the next item is inside an iframe, it consumes the `focusin` event, and therefore `.keyboard-outline` is not removed from the previous element.

**Comment by kanterjoe** [comment]
I have a fix to this issue that I can push, if you allow me to push.

**Comment by tech55541** [comment]
@kanterjoe Could you please make a PR?

**Comment by kanterjoe** [comment]
I believe that I need to be added as a contributor?

**Comment by tech55541** [comment]
@kanterjoe Please fork the repo in to your account and then make a PR from your repo to this repo.

https://help.github.com/en/articles/fork-a-repo

Thanks.

**Comment by kanterjoe** [comment]
Thanks. PR'd
